admin/includes/modules/preview_info.php

```diff
@@ -22,6 +22,14 @@
       $url = str_replace(['http://', 'https://'], '', $url);
   }
   unset ($url);
+  $p_date_available = $pInfo->products_date_available;
+  if (DATE_FORMAT_DATE_PICKER !== 'yy-mm-dd' && !empty($pInfo->products_date_available)) {
```

Comment:
Is the `!== 'yy-mm-dd'` because of a default db date format?

Reply:
Oh. I think I get it: this converts it TO y/m/d if not already in that format, to match MySQL standard date storage?